### PR TITLE
Added custom loss functions 

### PR DESCRIPTION
**Pull request by sushantag9**
Added the implementation of 
- Dice Coefficient
- Dice Coefficient Loss
- Soft Dice Loss

Updated the READMEM file

### CODE REVIEW

**Comment by varungupta31** [comment]
@sushantag9 Requesting you kindly take a look at my [issue#367](https://github.com/divamgupta/image-segmentation-keras/issues/367#issue-1206193159).
Thank you.

**Comment by sushantag9** [comment]
> @sushantag9 Requesting you kindly take a look at my [issue#367](https://github.com/divamgupta/image-segmentation-keras/issues/367#issue-1206193159). Thank you.

I will look into it and get back soon.
